src/Amazon.SQS.ExtendedClient/AmazonSQSExtendedClientBase.cs

```diff
@@ -459,5 +459,7 @@ public Task<UntagQueueResponse> UntagQueueAsync(UntagQueueRequest request, Cance
         }
 
         public IClientConfig Config => amazonSqsToBeExtended.Config;
+
+        public ISQSPaginatorFactory Paginators => throw new NotImplementedException();
```

Comment:
Is this being used anywhere? Don't we need to implement this?

Reply:
I don't see it being used anywhere and after searching for quite some time, I can't find good documentation on it or implementation that uses it.
The only thing I can find is that some of the methods like ListDeadLetterSourceQueues support pagination and if it isn't set, up to 1000 results will be retrieved.

Reply:
I have now tried just returning the same property from the injected IAmazonSQS object.